includes/wikia/parser/templatetypes/TemplateTypesParser.class.php

```diff
@@ -37,6 +38,36 @@ public static function onFetchTemplateAndTitle( &$text, &$finalTitle ) {
 	}
 
 	/**
+	 *
+	 * @param string $text - template content
+	 * @param Title $finalTitle - template title object
+	 *
+	 * @return bool
+	 */
+	public static function onFetchTemplateAndTitleForTables( &$text, &$finalTitle ) {
+		wfProfileIn( __METHOD__ );
```

Comment:
Please add new `wg` flag for this feature sot it could be easily turned on/off for each community